draft-ietf-spice-sd-cwt.md

```diff
@@ -731,6 +740,52 @@ Contact: Orie Steele (orie@transmute.industries)
 
 Security considerations from COSE {(RFC9052)} and CWT {{RFC8392}} apply to this specification.
 
+## Transparency
+
+Verification of an SD-CWT requires that the verifier have access to a verification key (public key) that is associated with the issuer.
+Compromise of the issuer's signing key enables an attacker to forge credentials for any subject associated with the issuer.
+Certificate transparency as described in {{-CT}}, or key transparency as described in {{-KT}} can enable the observation of missued certificates or fraudulent bindings between verification keys and issuer identifiers.
+Issuer's choose which claims to include in an SD-CWT, and whether they are mandatory to disclose, including self asserted claims such as "iss".
+All mandatory to disclose data elements are visible to the verifier as part of verification, some of these elements reveal information about the issuer, such as key or certificate thumbprints, supported digital signature algorithms, or operational windows which can be inferred from analysis of timestamps.
+
+## Traceability
+
+Presentations of the same SD-CWT to multiple verifiers can be correlated by matching on the signature component of the COSE_Sign1.
+Signature based linkability can be mitigated by leveraging batch issuance of single use tokens, for a credential management complexity cost.
+Any claim value with sufficiently low anonymity set can be used track the subject.
+For example, a high precision issuance time might match the issuance of only a few credentials for a given issuer, and as such any presentation of a credential issued at that time can be determined to be associated with the set of credentials issued at that time, for those subjects.
+
+## Credential Types
+
+The mandatory and optional to disclose data elements in an SD-CWT are credential type specific.
+Several distinct credential types might be applicable to a given use case.
+Issuer's MUST perform a privacy and confidentiality assessment regarding each credential type they intend to issue prior to issuance.
```

Comment:
```suggestion
Issuers MUST perform a privacy and confidentiality assessment regarding each credential type they intend to issue prior to issuance.
```

Reply:
Line 762 is incorrect. 

Issuers cannot do this. They should only do this for the credential types that are mandatory to be disclosed. 

Change into:

> For each credential type that will be mandatorily diclosed, prior to issuance, Issuers MUST perform a privacy and confidentiality assessment.

Reply:
@Denisthemalice I think you are confusing "credential type" (a profile with guidance about specific claims), and a claim. 

Credential types aren't disclosed, claims are.